.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
```

Comment:
Suggestion: CI is rather vague name for a workflow. What about "Build and deploy"? The name of the file should be renamed accordingly.